ProteoformSuiteGUI/ExperimentTheoreticalComparison.cs

```diff
@@ -80,6 +76,12 @@ public List<DataGridView> GetDGVs()
 
         public void ClearListsTablesFigures(bool clear_following)
         {
+            //clear all save acceptance actions --> will re-add save actions from loaded actions if peak still exists
+            Sweet.save_actions.RemoveAll(x => x.StartsWith("accept ExperimentalTheoretical") || x.StartsWith("unaccept ExperimentalTheoretical"));
+            //if in this round or others haven't ever shifted a mass, clear them all. Need to be careful because rerun the gamut whenever shift peaks.
+            if (!Sweet.lollipop.raw_experimental_components.Any(c => c.manual_mass_shift > 0))
```

Comment:
This would only clear them if no shifts ever occurred because the raw exp comps manual mass shift retains that value unless you refreshed load decon components